_data/authors.yml

```diff
@@ -190,6 +190,12 @@
   description: belongs to the epagesdevs content team.
   image: avatar_male.jpg
 
+- id: gsanemeterio
+  name: German
+  full_name: German
+  description: Here goes the description.
```

Comment:
Suggestion: is a JavaScript Developer who is now into Ruby on Rails. He is fond of video games, as well as ...

> Would like make an addition here. What are you specifically fond of? Any technologies you also like?

Reply:
I need 

> Suggestion: is a JavaScript Developer who is now into Ruby on Rails. He is fond of video games, as well as ...
> 
> > Would like to make an addition here. What are you specifically fond of? Any technologies you also like?

In 3rd person?

Reply:
Whatever you prefer. I just need some hints, and I'll shape it for you.